python/Ganga/GPIDev/Lib/GangaList/GangaList.py

```diff
@@ -52,8 +52,8 @@ def makeGangaListByRef(_list, preparable=False):
     result = GangaList()
     if len(_list) == 0:
         return result
-    temp_list = [stripProxy(element) for element in _list]
-    result._list = temp_list
+
+    result._list.extend([stripProxy(element) for element in _list])
```

Comment:
In that case how does the GangaList know that something else hasn't changed the objects it's supposed to be responsible for? How can we prevent someone deleting an element elsewhere without this class still having a copy as you would expect.